src/ontoweaver/tabular.py

```diff
@@ -117,26 +117,18 @@ def source_id(self, i, row):
 
 
     def properties(self, row, type):
-        """Extract properties of `type` from `row`."""
+        """Extract properties of each property category for the given node type. If no properties are found, return an empty dictionary."""
+
         properties = {}
-        # FIXME HERE properties for source node are not pulled.
-
-        # Find first matching parent class.
-        matching_class = None
-        for parent in type.mro(): # mro is guaranted in resolution order.
-            if parent in self.properties_of:
-                matching_class = parent
-                break
-        if not matching_class:
-            return {} # Defaults to no property.
-
-        # Extract and map the values.
-        for in_prop in self.properties_of[matching_class]:
-            out_prop = self.properties_of[type.__name__][in_prop]
-            properties[out_prop] = row[in_prop]
 
-        return properties
+        if type.__name__ in self.properties_of:
+            for key, value in self.properties_of[type.__name__].items():
+                if self.skip(row[key]):
+                    continue
+                else:
+                    properties[value] = str(row[key]).replace("'", "`") #FIXME enforce that its a string or replace single quotes... this works but is weird
```

Comment:
Why is the replace needed exactly?

Reply:
When importing into a Neo4j database, the presence of `'` is creating problems because Neo4j recognizes it as the end of the string. I also tried using `repr()` , but the only way I managed to import the data without any problems is completely removing or replacing  `'` characters.